SuiteSparse/.gitignore

```diff
@@ -0,0 +1,220 @@
+SPQR/Demo/C.mtx
+SPQR/Demo/E.txt
+SPQR/Demo/R.mtx
+SPQR/Demo/X.mtx
+SPQR/Demo/gpu_results.txt
+SPQR/Demo/qrdemo_gpu
+SPQR/Demo/qrdemo_gpu2
+SPQR/Demo/qrdemo_gpu3
+UMFPACK/Demo/numeric.umf
+UMFPACK/Demo/symbolic.umf
+UMFPACK/Demo/umfpack_di_demo
+UMFPACK/Demo/umfpack_dl_demo
+UMFPACK/Demo/umfpack_simple
+UMFPACK/Demo/umfpack_zi_demo
+UMFPACK/Demo/umfpack_zl_demo
+
+# ignore these specific programs in the Package/Tcov directories
+CHOLMOD/Tcov/cl
+CHOLMOD/Tcov/clread
+CHOLMOD/Tcov/cm
+CHOLMOD/Tcov/cmread
+CHOLMOD/Tcov/covs.out
+CHOLMOD/Tcov/ldemo
+CHOLMOD/Tcov/ldemo.c
+CHOLMOD/Tcov/temp*.mtx
+CHOLMOD/Tcov/timelog.m
+CHOLMOD/Tcov/l_*.c
+CHOLMOD/Tcov/z_*.c
+CHOLMOD/Tcov/zz_*.c
+CHOLMOD/Tcov/zl_*.c
+CHOLMOD/Tcov/zdemo
+CHOLMOD/Tcov/zdemo.c
+
+CSparse/Tcov/cov.out
+CSparse/Tcov/cov.sort
+CSparse/Tcov/cover.out
+CSparse/Tcov/covs.out
+CSparse/Tcov/cs_*.c
+CSparse/Tcov/cstcov_test
+CSparse/Tcov/*.out
+CSparse/Tcov/cs_demo1
+CSparse/Tcov/cs_demo2
+CSparse/Tcov/cs_demo3
+
+CXSparse/Tcov/cov.out
+CXSparse/Tcov/cov.sort
+CXSparse/Tcov/cover.out
+CXSparse/Tcov/covs.out
+CXSparse/Tcov/cs_*.c
+CXSparse/Tcov/cs*_ci.c
+CXSparse/Tcov/cs*_cl.c
+CXSparse/Tcov/cs*_di.c
+CXSparse/Tcov/cs*_dl.c
+CXSparse/Tcov/*.out
+CXSparse/Tcov/cs_demo1_ci
+CXSparse/Tcov/cs_demo1_cl
+CXSparse/Tcov/cs_demo1_di
+CXSparse/Tcov/cs_demo1_dl
+CXSparse/Tcov/cs_demo2_ci
+CXSparse/Tcov/cs_demo2_cl
+CXSparse/Tcov/cs_demo2_di
+CXSparse/Tcov/cs_demo2_dl
+CXSparse/Tcov/cs_demo3_ci
+CXSparse/Tcov/cs_demo3_cl
+CXSparse/Tcov/cs_demo3_di
+CXSparse/Tcov/cs_demo3_dl
+CXSparse/Tcov/cs_idemo
+CXSparse/Tcov/cs_ldemo
+CXSparse/Tcov/cstcov_test_ci
+CXSparse/Tcov/cstcov_test_cl
+CXSparse/Tcov/cstcov_test_di
+CXSparse/Tcov/cstcov_test_dl
+
+KLU/Tcov/cov_*.c
+KLU/Tcov/klutest
+KLU/Tcov/klultest
+KLU/Tcov/*.out
+
+SPQR/Tcov/X.mtx
+SPQR/Tcov/gpu_results.txt
+SPQR/Tcov/gpuqrengine_demo
+SPQR/Tcov/qrdemo_gpu
+SPQR/Tcov/qrtest
+SPQR/Tcov/qrtest_out.txt
+SPQR/Tcov/troll.m
+SPQR/Tcov/cov.out
+
+UMFPACK/Tcov/covall_err.out
+UMFPACK/Tcov/cover.out
+
+# ignore these specific files in the Package/MATLAB directories
+MATLAB_Tools/spqr_rank/save_samples_demo_spqr_rank.mat
+CXSparse/MATLAB/CSparse/cs_cl_*.c
+CXSparse/MATLAB/Test/cs_*.c
+
+RBio/Tcov/RBdemo
+RBio/Tcov/RBdemo.c
+RBio/Tcov/RBio.c
+RBio/Tcov/RBio.h
+RBio/Tcov/RBtest
+RBio/Tcov/SuiteSparse_config.c
+RBio/Tcov/SuiteSparse_config.h
+RBio/Tcov/*.out
+RBio/Tcov/*.rb
+
+# GraphBLAS
+GraphBLAS/Demo/bfs_demo.out
+GraphBLAS/Demo/complex_demo.m
+GraphBLAS/Demo/mis_demo.out
+GraphBLAS/Demo/simple_demo.out
+GraphBLAS/Demo/wildtype_demo.out
+GraphBLAS/Demo/tri_demo.out
+GraphBLAS/Test/errlog.txt
+GraphBLAS/Test/log.txt
+GraphBLAS/Doc/GraphBLAS_UserGuide.out
+GraphBLAS/Tcov/errlog.txt
+GraphBLAS/Tcov/log.txt
+GraphBLAS/Tcov/gbstat.mat
+
+tri/tri_main
+ktruss/allktruss_graphblas_main
+ktruss/allktruss_main
+ktruss/ktruss_graphblas_main
+ktruss/ktruss_main
+
+ssget/files/ss_index_old.mat
+ssget/files/ssstats_old.csv
+
+# Do not ignore this file
+!.gitignore
```

Comment:
why can't you just ignore SuiteSparse (and then not need this file)?

Reply:
Cheers for noticing this. I did add SuiteSparse to the gitignore so I am not sure why this gitignore has been added